noble/noble.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for noble
 // Project: https://github.com/sandeepmistry/noble
 // Definitions by: Seon-Wook Park <https://github.com/swook>
+// Additions by: Hans Bakker <https://github.com/wind-rider>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
I cannot find any other definition file which refers to an `../node/index.d.ts` and as far as I know the node definitions should be in `../node/` with version number sufficies.

I would suggest reverting to `../node/node.d.ts` as this works for both the CI tool as well as a clean setup.